OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -16,6 +16,6 @@
     <PackageReference Include="Mono.NAT" Version="3.0.1" />
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
-    <PackageReference Include="System.Threading.Channels" Version="5.0.0" />
+    <PackageReference Include="System.Threading.Channels" Version="6.0.0-rc.1.21451.13" />
```

Comment:
Is it safe to bump this one before we switch to net6?

Reply:
@teinarss ?

Reply:
This is going to need another update in a month when .NET 6 releases and is also an experimental build. I say skip it until the final thing is available.